src/components/connection-prompt/usb/manual/ManualInstallTutorial.svelte

```diff
@@ -9,16 +9,21 @@
   import { onMount } from 'svelte';
   import StandardButton from '../../../StandardButton.svelte';
   import Microbits from '../../../../script/microbit-interfacing/Microbits';
-  import ImageSkeleton from '../../../skeletonloading/ImageSkeleton.svelte';
   import Bowser from 'bowser';
   import transferFirmwareMacOSImage from '../../../../imgs/transfer_firmware_macos.gif';
   import transferFirmwareChromeOSImage from '../../../../imgs/transfer_firmware_chromeos.gif';
   import transferFirmwareWindowsImage from '../../../../imgs/transfer_firmware_windows.gif';
   import DialogHeading from '../../../DialogHeading.svelte';
+  import EscapedChunksFormattedMessage, {
```

Comment:
I don't think `EscapedChunksFormattedMessage` is used or defined?

Reply:
48f6c50b09be33c24a00448ba8941dbdd6b71d09